cts/cts-scheduler.in

```diff
@@ -1250,13 +1251,17 @@ class CtsScheduler(object):
         self.summary_dir = os.path.join(self.args.io_dir, "summary")
         self.stderr_expected_dir = os.path.join(self.args.io_dir, "stderr")
 
+        # Create a temproary directory to store diff file
+        self.failed_dir = tempfile.mkdtemp()
```

Comment:
Let's use prefix="cts-scheduler." so that if someone lists /tmp, they remember where the directory came from